Ally4/Property-Pro-Lite
# #167335779 The operator should be able to set the working environment

#### What does this PR do?
This PR is for the setting of the working environment for challenge 3
#### Description of Task to be completed?
Creation of develop2 as a branch from master
Pushing develop2 on the Github
Make a pull request on develop2 from develop
Merging develop to develop2
#### How should this be manually tested?
It can be tested by going on the local repository
And check into the branches 
Then select develop2
#### Any background context you want to provide?
No,
#### What are the relevant pivotal tracker stories?
#167335779
